todo_app/app.py

```diff
@@ -1,21 +1,25 @@
 from flask import Flask
 
 # we need to import this method for rendering templates and is not imported by default
-from flask import render_template  , redirect
+from flask import render_template , redirect
 
 # we need request module to query the application from browser
 from flask import request
+import os
+
+# Extract the ENV variables 
+url = os.getenv('URL')
+apikey = os.getenv('APIKEY') 
+apitoken = os.getenv('APIToken')
+trellobid = os.getenv('TrelloBID')
 
 ########################################################################################
 #####  The default repo has some functions written in ./data/sessions_items.py
 ####   Importing functions written under ./data/sessions_items.py
 #####
-from todo_app.data.session_items import get_items
-from todo_app.data.session_items import get_item
-from todo_app.data.session_items import add_item
-from todo_app.data.session_items import save_item
-from todo_app.data.session_items import remove_item   # I have added this for strech goal
-
+from todo_app.data.trello_items import get_items
+from todo_app.data.trello_items import add_item
+from todo_app.data.trello_items import update_item
```

Comment:
Minor and just a preference, but note that you can collect these all together onto one line:
```suggestion
from todo_app.data.trello_items import get_items, add_item, update_item
```